zephyr/Kconfig

```diff
@@ -12,12 +12,26 @@ config WPA_SUPP
     select NET_SOCKETS_PACKET
     select NET_SOCKETPAIR
     select NET_L2_WIFI_MGMT
+    # For newlib min HEAP check, NCS_SAMPLE_DEFAULTS doesn't print assert information
+    select LOG_DEFAULT_MINIMAL
+	select ASSERT
+    select ASSERT_VERBOSE
+    # Without this reset on fatal error is triggered
+    select DEBUG
+    ###
     select EXPERIMENTAL if !SOC_NRF5340_CPUAPP_QKAA
     help
       WPA supplicant implements 802.1X related functions.
 
 if WPA_SUPP
 
+# For asserts, this avoids continuous reset
+config RESET_ON_FATAL_ERROR
+    default n
+
+config NEWLIB_LIBC_MIN_REQUIRED_HEAP_SIZE
```

Comment:
Just curious, why aren't we using zephyr system heap? How will this work if an application wants to use a differnt C library, PICOLIB for instance.